test/cluster/object_store/test_backup.py

```
@@ -1805,3 +1822,95 @@ async def test_cluster_snapshot_repair_set_unique(manager: ManagerClient, object
+    prefix = "smurfette"
+    await manager.api.enable_injection(g0c.ip_addr, breakpoint_name, one_shot=False)
+
+    backup_tid = await manager.api.backup_cluster_snapshot(s.ip_addr, ks, snapshot_name, s.datacenter, 
+                                                           object_storage.address, object_storage.bucket_name,
+                                                           prefix, tables = [cf])
+
+    await manager.api.wait_for_injection_enter(g0c.ip_addr, breakpoint_name)
+
+    deadline = time.time() + 60.0
+
+    async def _check():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_check`

Add return type annotation: `bool | None`

(ANN202)